pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PqlQueryResource.java

```diff
@@ -108,6 +108,8 @@ public String getQueryResponse(String pqlQuery, String traceEnabled, String quer
     BrokerRequest brokerRequest;
     try {
       brokerRequest = REQUEST_COMPILER.compileToBrokerRequest(pqlQuery);
+      String inputTableName = brokerRequest.getQuerySource().getTableName();
```

Comment:
Unrelated to this PR but may be at some point we need to fix this?

Just to extract the table name, a query received on the controller via rest API will be compiled to BrokerRequest twice -- once at the controller and then at the broker.

Reply:
yes, I realized that as well. Query console is generally used for debugging and not worth optimizing it.